website/components/typeItAnimation.js

```diff
@@ -0,0 +1,9 @@
+import TypeIt from "typeit-react"
+
+export default function TypeItAnimation() {
+    return (
+        <div>
+             <TypeIt>nfddfdfdfdfdffd</TypeIt>
+        </div>
+    )
+}
```

Comment:
Let's add new line in the end of the file

Reply:
done